{"name": "Test login with incorrect login credentials with parameters -- @1.1 Credentials", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps_with_parameters.py\", line 13, in step_impl\n    SeleniumHelper(context.driver).open_page(LOGIN_URL)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "When user input wrong \"zapel\" and password \"zapel\"", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps_with_parameters.py\", line 13, in step_impl\n    SeleniumHelper(context.driver).open_page(LOGIN_URL)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "attachments": [{"name": ".text", "source": "927f3647-3cab-4869-ad8d-7be226ec20bd-attachment.txt", "type": "text/plain"}], "start": 1727202851949, "stop": 1727202851949}, {"name": "Then error message will come with parameters", "status": "skipped", "start": 1727202851949, "stop": 1727202851949}], "parameters": [{"name": "login_id", "value": "zapel"}, {"name": "password", "value": "zapel"}], "start": 1727202851948, "stop": 1727202851949, "uuid": "232ea4d5-404e-4e93-85ed-c5880184ad4a", "historyId": "957e6aa493a3bfb3ee142d79f26263a2", "fullName": "Facebook login functionality with parameters: Test login with incorrect login credentials with parameters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Facebook login functionality with parameters"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}